modules/debts_api/app/models/debts_api/v0/form5655_submission.rb

```diff
@@ -82,14 +82,21 @@ def set_vha_completed_state(status, options)
 
     def register_failure(message)
       failed!
-      if message.blank?
-        message = "An unknown error occurred while submitting the form from call_location: #{caller_locations&.first}"
-      end
+      message ||= "An unknown error occurred while submitting the form from call_location: #{caller_locations&.first}"
+
       update(error_message: message)
       Rails.logger.error("Form5655Submission id: #{id} failed", message)
+
       StatsD.increment("#{STATS_KEY}.failure")
-      alert_silent_error unless message.include?('SharepointRequest')
+
+      if message.to_s.include?('SharepointRequest')
+        Rails.logger.warn("SharePoint request failed with: #{message}")
+      else
+        alert_silent_error
```

Comment:
It seems like we don't log the error when we fire the `alert_slient_error` method.

I will comment where I mean.

Reply:
Am I missing where we do that? Like, where we could answer this question?

"We fired the alert_silent_error method with this error message"